auth_totp/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
```

Comment:
2016-2017 - we wrote this in both years, copyright is from first write point to last write point.